spec/models/rating_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Rating, type: :model do
   let(:user) { FactoryBot.create(:user) }
   let(:movie) { FactoryBot.create(:movie) }
-  let(:rating) { FactoryBot.build(:rating) }
+  let(:rating) { FactoryBot.build(:rating, user: user, movie: movie) }
```

Comment:
Must be a change to the way FactoryBot works. I didn't look into it, just wanted to get the tests to pass. When upgrading to Rails 5, some refactoring should be done here.

Reply:
Yes, this is a (breaking) change to the way FactoryBot works. I ran into this somewhat recently. Glad you caught it and upgraded it 👍